### PR TITLE
Close #32. Add Code Mirror mode and mime type info to languages

### LINKED ISSUE #32: Add Code Mirror modes to Languages

### PR DESCRIPTION
**Pull request by Glavin001**

### CODE REVIEW

File: src/options.ts [hunk 1]
```diff
@@ -268,7 +274,7 @@ export const Options: OptionsRegistry = {
   },
   space_before_conditional: {
     default: true,
-    description: "Add a space before conditional, \"if(true)\" vs \"if (true)\"",
+    description: 'Add a space before conditional, "if(true)" vs "if (true)"',
```
Comment (Glavin001) [inline src/options.ts]: Bug.

Comment (szeck87) [reply src/options.ts]: Did you beautify locally and that's what changed it?

Comment (Glavin001) [reply src/options.ts]: This is Pretty Diff bug. Locally I only use Prettier. We should change all `.unibeautifyrc.yml` files to use just Prettier until Pretty Diff is fixed.

Comment (Glavin001) [reply src/options.ts]: See https://github.com/prettydiff/prettydiff/issues/508

**Pull request merged**